@claude secator/cli.py:1384:121: E501 line too long (125 > 120 characters)
add a  # noqa: E501 at the end of the line (make sure you put two spaces before the '#')

--- a/secator/cli.py
+++ b/secator/cli.py
@@ -1381,7 +1381,7 @@ def report_info(runner_id, workspace, show_all):
 	if not runner_type.endswith('s'):
 		runner_type += 's'
 
-	report_path = Path(CONFIG.dirs.reports) / sanitize_folder_name(workspace_name) / runner_type / runner_number / 'report.json'
+	report_path = Path(CONFIG.dirs.reports) / sanitize_folder_name(workspace_name) / runner_type / runner_number / 'report.json'  # noqa: E501
 	if not report_path.exists():
 		console.print(Error(message=f'Report not found: {report_path}'))
 		return

